experimental/devices/pca9685/pins.go

```diff
@@ -0,0 +1,115 @@
+package pca9685
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"time"
+
+	"periph.io/x/periph/conn/gpio"
+	"periph.io/x/periph/conn/gpio/gpioreg"
+	"periph.io/x/periph/conn/physic"
+	gpiopin "periph.io/x/periph/conn/pin"
+)
+
+const (
+	dutyMax gpio.Duty = math.MaxUint16
```

Comment:
gpio.DutyMax ?

Reply:
this is the maximum duty value (16 bit) supported by pca9685. The 24 bit value used by the gpio api is scaled down to this. Should I rename this to `dutyMax16`?

Reply:
Oh right, ignore this.